swagger_spec_validator/common.py

```diff
@@ -30,10 +36,18 @@ def validate_json(json_document, schema_path):
     schema_path = resource_filename('swagger_spec_validator', schema_path)
     with open(schema_path) as schema_file:
         schema = json.loads(schema_file.read())
-    resolver = RefResolver('file://{0}'.format(schema_path), schema)
+    resolver = RefResolver(
+        'file://{0}'.format(schema_path),
+        schema,
+        store={'http://swagger.io/v2/schema.json': schema})
```

Comment:
sorry, I meant `store={schema['id']: schema}`

This should already be covered by tests. I'll double check that it works with 1.2.